algoliasearch-core/src/main/java/com/algolia/search/models/indexing/PartialUpdateOperation.java

```diff
@@ -0,0 +1,53 @@
+package com.algolia.search.models.indexing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PartialUpdateOperation<T extends Object> {
```

Comment:
No need to explicistly extend `Object`, we can replace `T extends Object` by `T`